documentation/configuration.md

```diff
@@ -79,14 +79,15 @@ config :crawly,
 
 ### middlewares :: [module()]
 
-```elixir
 The default middlewares are as follows:
```

Comment:
Should be "an example of middlewares", since it does not actually match the defaults used in worker

Reply:
agreed